portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -115,34 +93,88 @@ export default {
         }
       )
     },
-    isShowCategoryItem({ category, item, indexItem }) {
-      return (
-        !item.is_empty && (category.showAll || indexItem < this.visible_items)
-      )
-    },
     getTitleTranslation(category, language) {
-      if (
-        !_.isNil(category.title_translations) &&
-        !_.isEmpty(category.title_translations)
-      ) {
+      if (category && category.title_translations) {
         return category.title_translations[language]
       }
       return category.name
+    },
+    datesRangeFilter() {
+      if (this.selectedFilters) {
```

Comment:
Kan dit null zijn? Kunnen we het default een leeg array maken? Scheelt weer condities.

Reply:
✅